src/components/ProfileUpdate/index.tsx

```diff
@@ -6,7 +6,8 @@ import React, {
   ChangeEventHandler,
   FormEventHandler,
 } from 'react';
-import { Form, Input, Button, Select, Modal, Upload, Avatar } from 'antd';
+import { Form, Input, Button, Select, Modal, Avatar } from 'antd';
+import * as ANTD from 'antd';
```

Comment:
This is fine for now. The edit profile (and more specifically profile photo upload) got a bit messy with the TS rewrite so I think we can redo it at a later point.